seacrowd/sea_datasets/uit_vsfc/uit_vsfc.py

```diff
@@ -0,0 +1,207 @@
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "sentences_path": data_dir["train"]["sentences"],
+                    "sentiments_path": data_dir["train"]["sentiments"],
+                    "topics_path": data_dir["train"]["topics"],
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "sentences_path": data_dir["test"]["sentences"],
+                    "sentiments_path": data_dir["test"]["sentiments"],
+                    "topics_path": data_dir["test"]["topics"],
+                    "split": "test",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "sentences_path": data_dir["validation"]["sentences"],
+                    "sentiments_path": data_dir["validation"]["sentiments"],
+                    "topics_path": data_dir["validation"]["topics"],
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, sentences_path: Path, sentiments_path: Path, topics_path: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        if self.config.schema == "source":
+            with open(sentences_path, encoding="utf-8") as sentences, open(sentiments_path, encoding="utf-8") as sentiments, open(topics_path, encoding="utf-8") as topics:
+                for key, (sentence, sentiment, topic) in enumerate(zip(sentences, sentiments, topics)):
+                    yield key, {
+                        "sentence": sentence.strip(),
+                        "sentiment": int(sentiment.strip()),
+                        "topic": int(topic.strip()),
+                    }
+
+        elif self.config.name == f"{_DATASETNAME}_sentiment_seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            with open(sentences_path, encoding="utf-8") as sentences, open(sentiments_path, encoding="utf-8") as sentiments:
+                for key, (sentence, sentiment) in enumerate(zip(sentences, sentiments)):
+                    yield key, {"id": str(key), "text": sentence.strip(), "label": int(sentiment.strip())}
+        elif self.config.name == f"{_DATASETNAME}_topic_seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            with open(sentences_path, encoding="utf-8") as sentences, open(topics_path, encoding="utf-8") as topics:
+                for key, (sentence, topic) in enumerate(zip(sentences, topics)):
+                    yield key, {
+                        "id": str(key),
+                        "text": sentence.strip(),
+                        "label": int(topic.strip()),
+                    }
+
+
+if __name__ == "__main__":
+    datasets.load_dataset(__file__)
```

Comment:
don't forget to remove this ;)

and remove training lines after end of code

Reply:
I have pushed a new commit to remove those two lines. But sorry I don't quite understand the meaning of 'training lines after end of code'.

Reply:
^I think Sabil meant the training lines / whitespace :D

Reply:
Trailing lines, apologies for typo

Reply:
@sabilmakbar Have deleted and commit. Please have a check :)